src/main/java/core/basesyntax/dao/FruitShopDaoImpl.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.Operation;
+import core.basesyntax.service.OperationStrategy;
+import core.basesyntax.service.operations.OperationService;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+public class FruitShopDaoImpl implements FruitShopDao {
+    private OperationService operationService;
+
+    @Override
+    public void updateStorage(String data) {
+        List<String> dataToList = Arrays.asList(data.split(System.lineSeparator()));
+        dataToList.stream()
+                .map(s -> s.split(","))
```

Comment:
it's better to seprate logic that parses our data because our format of incoming data file may change and will have to change this logic here, but this class should be only responsible for working with storage, fix it in other methods as well